File: Source/AbidzarFrame.API.Service/App_Start/AuthenticationConfiguration.cs
using System.Configuration;

namespace AbidzarFrame.API.Service
{
    public class AuthenticationConfiguration : ConfigurationSection
    {
        public static AuthenticationConfiguration GetInstance()
        {
            AuthenticationConfiguration instance = null;
            if ((instance == null))
            {
                instance = ConfigurationManager.GetSection("AuthenticationConfiguration") as AuthenticationConfiguration;
            }

            return instance;
        }

        [ConfigurationProperty("ConfigFilePath", DefaultValue = "")]
        public string ConfigFilePath
        {
            get
            {
                return (string)this["ConfigFilePath"];
            }
            set
            {
                this["ConfigFilePath"] = value;
            }
        }
        [ConfigurationProperty("WebServiceAuthenticationToken", DefaultValue = "")]
        public string WebServiceAuthenticationToken
        {
            get
            {
                return (string)this["WebServiceAuthenticationToken"];
            }
            set
            {
                this["WebServiceAuthenticationToken"] = value;
            }
        }

        [ConfigurationProperty("WebServiceAuthenticationDB", DefaultValue = "")]
        public string WebServiceAuthenticationDB
        {
            get
            {
                return (string)this["WebServiceAuthenticationDB"];
            }
            set
            {
                this["WebServiceAuthenticationDB"] = value;
            }
        }

    }
}

File: Source/AbidzarFrame.WCF.Service/AuthenticationConfiguration.vb
Imports System.Configuration

Public Class AuthenticationConfiguration
    Inherits ConfigurationSection

    Protected Shared instance As AuthenticationConfiguration = Nothing

    Public Shared Function GetInstance() As AuthenticationConfiguration
        If (instance Is Nothing) Then
            instance = CType(ConfigurationManager.GetSection("AuthenticationConfiguration"), AuthenticationConfiguration)
        End If
        Return instance
    End Function

    <ConfigurationProperty("Token", DefaultValue:="")>
    Public Property Token As String
        Get
            Return Me("Token")
        End Get
        Set(ByVal value As String)
            Me("Token") = value
        End Set
    End Property

End Class
